src/router_node.c

```diff
@@ -226,9 +227,22 @@ static void AMQP_rx_handler(void* context, qd_link_t *link, pn_delivery_t *pnd)
     //        send it.
     //
     msg = qd_message_receive(pnd);
+
     if (!msg)
         return;
 
+
+    char repr[qd_message_repr_len()];
```

Comment:
This declaration should be inside the if statement.  You are needlessly allocating stack space when logging is disabled.